src/dom.js

```diff
@@ -1095,7 +1095,7 @@ export function getShadowHost(element : Node) : ?HTMLElement {
     const shadowRoot = getShadowRoot(element);
 
     // $FlowFixMe
-    if (shadowRoot.host) {
+    if (shadowRoot && shadowRoot.host) {
```

Comment:
Does this change allow us to remove the `// $FlowFixMe` comment above?

Reply:
actually no, even with that extra validation, that comment is needed